doc/security-warnings.md

```diff
@@ -44,3 +43,8 @@ REST Interface
 The REST interface is a feature inherited from upstream Bitcoin.  By default,
 it is disabled. We do not recommend you enable it until it has undergone a
 security review.
+
+Confirmations and Reorgs
+----------------------------
+
+When using a low number of minimum confirmations, users should be aware of new behavior in Zcash that differs significantly from Bitcoin. In the case of a blockchain reorganization, Bitcoin's coinbase maturity helps ensure that any reorg shorter than the maturity interval will not invalidate any of the rolled-back transactions. However for Zcash, all JoinSplits which were anchored within the reorg interval will become invalid, rolling back transactions and reverting funds to the original owner. The transaction rebroadcast mechanism inherited from Bitcoin will not successfully rebroadcast transactions containing JoinSplits if the anchor needs to change--the JoinSplit creator must do that.
```

Comment:
Nits:
* s/coinbase maturity/coinbase maturity rule/
* s/blockchain/block chain/
* use a Unicode em-dash in place of ``--``.